### PR TITLE
Create a page to Enter OpenAI API Key

### PR DESCRIPTION
**Pull request by niimurank**

### CODE REVIEW

**Comment by k-taro56** [comment]
Thank you for your Contribution!
The filename should be KebabCase. I will make the change this time.

File: src/components/SessionStorageViewer.tsx
Comment (k-taro56) [inline src/components/SessionStorageViewer.tsx:1]: Is this file for Debugging?

Comment (k-taro56) [reply src/components/SessionStorageViewer.tsx:1]: OK. I will make a debug branch and remove this change from this pull request.

**Review by umiyuri777 — APPROVED** [review]

File: src/components/SessionStorageViewer.tsx
Comment (k-taro56) [reply src/components/SessionStorageViewer.tsx:1]: #80

**Pull request merged**